src/main/webapp/populateData.js

```diff
@@ -162,6 +165,22 @@ function populateCalendar() {
 }
 
 /**
+ * Convert the time in hours and minutes,
+ * and concatenate the string to be displayed
+ * @param {int} timeMilli the time in milliseconds that needs to be converted
+ * @return {String} String representing the converted free time
+ */
+function convertTime(timeMilli) {
```

Comment:
It may be simpler to have the servlet provide the data in separate hours and minutes. At least that way you could unit test this logic.

Reply:
`Resolved`